exchange/src/bots/mm.hpp

```diff
@@ -0,0 +1,35 @@
+
+#include <vector>
+
+namespace nutc {
+namespace bots {
+
+class MarketMakerBot {
+public:
+    float long_positions;
+    float short_positions;
+
+    float long_capital;
+    float short_capital;
+
+    float capital_limit;
+
+    // TODO: fix input
+    explicit inline MarketMakerBot() {}
+    // TODO: implement
+    std::vector<float> take_action(float new_theo);
+private:
+    inline float compute_net_exposure() {
+        return (long_positions - short_positions);
+    }
+    inline float compute_capital_util() {
+        return (long_capital + short_capital)/capital_limit;
+    }
+    inline float compute_capital_tolerance() {
+        return (-1 * compute_net_exposure()) * (1 + (1 - compute_capital_util()) * capital_limit/2);
+    }
```

Comment:
postfix with `_` for private methods

also you don't need inline if its in a class